### ISSUE TITLE
Is exec function required?

### ISSUE THREAD

**Issue by d47081** [ygguser/simple_sites_directory#7]
I've just discovered build-in [PHP function](https://www.php.net/manual/ru/function.dns-get-record.php) `dns_get_record`

Is there any reason to use `dig` with `shell_exec` parse instead?

**Comment by ygguser** [comment]
Yes, for some unknown reason, this function returns an empty array...
```
$ cat dns.php
<?php
    $servers = array('302:db60::53 ', '301:1088::53 ');
    $result = dns_get_record("sites.ygg",  DNS_A6, $servers);
    print_r($result);
?>
```
```
$ php -f dns.php 
Array
(
)
```

**Comment by ygguser** [comment]
I haven't researched the php sources thoroughly, but I can assume that addresses from the deprecated range are filtered out there...

I am using php 7.4 from the official Debian repository.
```
# php -i |grep IPv6
IPv6 Support => enabled
```
